composer.json

```
@@ -10,7 +10,7 @@
     }
   ],
   "require": {
-    "php": "~8.1.0",
+    "php": "~8.1.0||~8.2.0||~8.3.0",
```

While this is common in the world of Magento extensions for some reason, it is not necessary, nor adivsable, to lock down the PHP version always only to the current stable feature version. This should bechanged to

```suggestion
    "php": "^8.1",
```

instead.

---

@fritzmg You proposition also contain PHP  >= 8.4 which could introduce some deprecations.

---

> You proposition also contain PHP >= 8.4 

That is correct and on purposes.

>  which could introduce some deprecations.

Which is fine.